layout.html

```diff
@@ -16,7 +16,7 @@
   <body>
 
     <nav class="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0">
-      <a class="navbar-brand col-sm-3 col-md-2 mr-0" href="##BASE_URL##">Fuse: Documentation</a>
+      <a class="navbar-brand col-sm-3 col-md-2 mr-0" href="##BASE_URL##">Fuse Open Documentation</a>
```

Comment:
I think this perhaps deserves its own commit. Nice catch!

Reply:
Ok